tools/lint

```diff
@@ -94,6 +94,9 @@ def run() -> None:
                                   description="Syntactic Grep (semgrep) Code Search Tool "
                                   "(config: ./tools/semgrep.yml)")
 
+    linter_config.external_linter('thirdparty', ['tools/check-thirdparty'],
+                                  description="Check docs/THIRDPARTY copyright file syntax")
```

Comment:
How long does this take to run?  If it's material, we can probably add options to have this not run by default except in CI.

Reply:
0.2 seconds.

Reply:
Make that 0.1 seconds (it was previously spending half its time walking through `static/generated/emoji` for no real reason).